File: js/src/test/Exchange/test.fetchBorrowInterest.js
import testBorrowInterest from './base/test.borrowInterest.js';
import testSharedMethods from './base/test.sharedMethods.js';
async function testFetchBorrowInterest(exchange, skippedProperties, code, symbol) {
    const method = 'fetchBorrowInterest';
    const borrowInterest = await exchange.fetchBorrowInterest(code, symbol);
    testSharedMethods.assertNonEmtpyArray(exchange, skippedProperties, method, borrowInterest, code);
    for (let i = 0; i < borrowInterest.length; i++) {
        testBorrowInterest(exchange, skippedProperties, method, borrowInterest[i], code, symbol);
    }
}
export default testFetchBorrowInterest;

File: julia/Ccxt/test/exchange/test.fetchBorrowInterest.jl
using Test
using Ccxt
function testFetchBorrowInterest(exchange, skippedProperties, code, symbol)

    method = "fetchBorrowInterest";
    borrowInterest = Base.fetch(fetchBorrowInterest(exchange, code, symbol));
    assertNonEmtpyArray(testSharedMethods, exchange, skippedProperties, method, borrowInterest, code);
    i = 0
    while functions.ccxtruthy(functions.ccxt_lt(i, length(borrowInterest)))
        testBorrowInterest(exchange, skippedProperties, method, get(borrowInterest, i + 1, nothing), code, symbol);
        i += 1
    end
    return true
end
